src/common/pqclean_shims/crypto_declassify.h

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: CC0
```

Comment:
Is CC0 really right/necessary here? This adds (complication) to #1437...

Reply:
c046b7553f4839ffb0ee745606792bddc203cbf5 replaces `crypto_declassify.h` with a MIT version.